dali/operators/util/normal_distribution_op.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/util/normal_distribution_op.h"
+#include "dali/core/static_switch.h"
+
+namespace dali {
+
+
+DALI_SCHEMA(NormalDistribution)
+                .DocStr(R"code(Creates a tensor that consists of data distributed normally.
+This operator can be ran in 3 modes, which determine the shape of the output tensor:
+1. Providing an input batch to this operator resolves in a batch of output tensors,
+where the shapes correspond to the shapes of tensors in the input batch.
+2. Providing a custom shape as an argument resolves in a batch of output tensors,
+where every output tensor has the same shape.
+3. Providing no input arguments resolves in a batch of output scalars,
+distributed normally (similar to `CoinFlip` operator).)code")
```

Comment:
- Please consider my suggestion about the wording above.
- I'd drop the CoinFlip reference here. It'd only confuse the user as CoinFlip is not a normal distribution

Reply:
Removed CionFlip reference